src/components/pages/Index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+
+const Index = () => {
```

Comment:
The landing page is the Dashboard. I dont think this index file should be included in this PR